src/main/java/org/threadly/litesockets/udp/UDPClient.java

```diff
@@ -0,0 +1,118 @@
+package org.threadly.litesockets.udp;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.net.SocketAddress;
+import java.nio.ByteBuffer;
+import java.nio.channels.SocketChannel;
+
+import org.threadly.concurrent.SubmitterExecutorInterface;
+import org.threadly.litesockets.Client;
+
+public class UDPClient extends Client {
+  protected final SocketAddress sa;
+  protected final UDPServer udpServer;
+  
+  protected UDPClient(SocketAddress sa, UDPServer server) {
+    this.sa = sa;
+    udpServer = server;
+  }
+  
+  @Override
+  public boolean equals(Object o) {
+    if(o instanceof UDPClient) {
+      if(hashCode() == o.hashCode()) {
+        UDPClient u = (UDPClient)o;
+        if(u.sa.equals(this.sa) && u.udpServer.getSelectableChannel().equals(udpServer.getSelectableChannel())) {
+          return true;
+        }
+      }
+    }
+    return false;
+  }
+  
+  @Override
+  public int hashCode() {
+    return sa.hashCode() * udpServer.getSelectableChannel().hashCode();
+  }
+  
+  @Override
+  protected void setThreadExecuter(SubmitterExecutorInterface sei) {
```

Comment:
I needed to expose it so UDPServer could access it since the UDPServer has to fake some of the server side stuff it needs to act in part like how SocketExecuter acts for tcpServer.